courses/management/commands/sync_grades_and_certificates.py

```diff
@@ -102,7 +102,15 @@ def handle(
                 course_run_grade.passed = bool(override_grade)
                 course_run_grade.letter_grade = None
                 course_run_grade.set_by_admin = True
-                course_run_grade.save_and_log(None)
+                try:
```

Comment:
Instead of wrapping `course_run_grade.save_and_log(None)` with try/except, I think we should add the whole thing under the try except, So that any failures during the creation of a certificate can be ignored.

With this suggestion, The try block would start before https://github.com/mitodl/mitxpro/blob/master/courses/management/commands/sync_grades_and_certificates.py#L93

and end after https://github.com/mitodl/mitxpro/blob/master/courses/management/commands/sync_grades_and_certificates.py#L109